openedx/core/djangoapps/content/block_structure/tests/helpers.py

```diff
@@ -1,14 +1,34 @@
 """
 Helpers for Course Blocks tests.
 """
+from openedx.core.lib.block_structure.exceptions import BlockStructureNotFound
+from openedx.core.lib.block_structure.store import BlockStructureStore
+
+from openedx.core.djangolib.testing.waffle_utils import override_switch
 
-from openedx.core.lib.block_structure.cache import BlockStructureCache
 from ..api import get_cache
+from ..config import _bs_waffle_switch_name
 
 
 def is_course_in_block_structure_cache(course_key, store):
     """
     Returns whether the given course is in the Block Structure cache.
     """
     course_usage_key = store.make_course_usage_key(course_key)
-    return BlockStructureCache(get_cache()).get(course_usage_key) is not None
+    try:
+        BlockStructureStore(get_cache()).get(course_usage_key)
+        return True
+    except BlockStructureNotFound:
```

Comment:
**Note to reviewers:** `BlockStructureStore.get` now raises an error instead of returning `None` when not found.